src/Tests/Microsoft.TemplateEngine.Cli.UnitTests/ParserTests/MiscTests.cs

```diff
@@ -23,9 +24,7 @@ public void KnownHelpAliasesAreCorrect()
 
             CliOption helpOption = result.CommandResult
                 .Children
-                .OfType<OptionResult>()
-                .Select(r => r.Option)
-                .Where(o => o.Aliases.Contains(Constants.KnownHelpAliases[0]))
+                .OfType<HelpOption>()
```

Comment:
since `Constants.KnownHelpAliases` was removed, this test can be removed.
wondering why this is not a compliation error 👀